README.md

```diff
@@ -1,2 +1,122 @@
-# melita-io
-Module for communication with melita.io API for devices
+# Melita IO
+
+|                |                            |
+| -------------- | -------------------------- |
+| Name           | Melita IO                  |
+| Version        | v1.0.0                     |
+| Dockerhub Link | [weevenetwork/melita-io]() |
+| Authors        | Mesud Pasic                |
+
+- [MQTT Ingress](#melita-io)
+  - [Description](#description)
+  - [Features](#features)
+  - [Environment Variables](#environment-variables)
+    - [Module Specific](#module-specific)
+    - [Set by the weeve Agent on the edge-node](#set-by-the-weeve-agent-on-the-edge-node)
+  - [Dependencies](#dependencies)
+
+## Description
+
+Melita.IO API module.
```

Comment:
Please give a more informative description, maybe something similar and broader to the description from YAML file ("Module for communication with melita.io API for devices controller, for sending requests to multiple devices")

Reply:
updated.